.travis.yml

```diff
@@ -26,17 +28,23 @@ install:
 
 jobs:
   include:
+    - stage: compile
+      name: "TypeScript compiler for Node"
+      node_js: 
```

Comment:
this is not running `node 10` anymore

Reply:
What is the correct syntax to run on both?

Reply:
Figured it out - reverted most of what i had and just added the compile in the existing stage.